presentation/src/main/java/com/dcs/presentation/ui/trend/TrendRoute.kt

```diff
@@ -218,33 +222,49 @@ fun TrendMovieUiStateScreen(
 }
 
 @Composable
-fun TrendMovieList(
-    movieItems: PagingData<MovieItemUiState>,
+fun TrendMovies(
+    movieItems: Flow<PagingData<MovieItemUiState>>,
     modifier: Modifier = Modifier,
 ) {
-    val pagingItems = snapshotFlow { movieItems }.collectAsLazyPagingItems()
+    val pagingItems = movieItems.collectAsLazyPagingItems()
     val listState = rememberLazyListState()
 
-    LazyRow(
-        modifier = modifier
-            .fillMaxWidth(),
-        state = listState,
-        contentPadding = PaddingValues(
-            horizontal = dimensionResource(id = R.dimen.list_margin_horizontal),
-            vertical = dimensionResource(id = R.dimen.list_margin_vertical)
+    val isLoading = pagingItems.loadState.refresh is LoadState.Loading
+    val isError = pagingItems.loadState.refresh is LoadState.Error
+    val isEmpty = pagingItems.itemCount == 0
+
+    when {
+        isLoading -> LoadingScreen()
+
+        isError -> ErrorScreen()
+
+        isEmpty -> ErrorScreen(
+            message = stringResource(id = R.string.empty_content_list_message)
         )
```

Comment:
```suggestion
        isEmpty -> {
            ErrorScreen(
                message = stringResource(id = R.string.empty_content_list_message)
            )
        }
```
한 줄로 처리가 안되면 brace 선언하는게 좋을 것 같아요.

Reply:
수정 완료 [ef3d03d](https://github.com/Edgar-J-Shin/AllAboutMovie/pull/29/commits/ef3d03da2154b6a0574f37a5152be506a27b1c7e)